likecoin/js/sidebar/src/index.js

```diff
@@ -210,7 +220,7 @@ function LikeCoinSideBar() {
         const redirectString = encodeURIComponent(siteurl);
         const titleString = encodeURIComponent(title);
         const fingerprint = fingerprints.join(',');
-        const popUpWidget = `https://like.co/in/widget/wordpress?fingerprint=${fingerprint}&publisher=&title=${titleString}&tags=${tagsString}&url=${urlString}&to=like-arweave&amount=${LIKE}&remarks=${memoString}&opener=1&redirect_uri=${redirectString}`;
+        const popUpWidget = `https://like.co/in/widget/wordpress?fingerprint=${fingerprint}&publisher=&title=${titleString}&tags=${tagsString}&url=${urlString}&to=like-arweave&amount=0&remarks=${memoString}&opener=1&redirect_uri=${redirectString}`;
```

Comment:
Why `0`?

Reply:
Since you don't need to do LIKE Pay again if you are at this step.